sdk/storage/azure-storage-blob/azure/storage/blob/_shared/response_handlers.py

```diff
@@ -22,6 +22,7 @@
 from .parser import _to_utc_datetime
 
 
+SV_DOCS_URL = "https://learn.microsoft.com/en-us/rest/api/storageservices/versioning-for-the-azure-storage-services"
```

Comment:
Link works now (from fixing the missing `s`) but need to omit `/en-us/` (as you had it previously) so that it can correctly select the right localization for our users. Not everyone will resolve to `en-us` 😉